### ISSUE TITLE
Remove TreeView code?

### ISSUE THREAD

**Issue by 3xtr4t3rr3str14l** [ChainSafe/simpleserialize.com#47]
Currently, the TreeView code is not visible on the site, but the code is still there.  It's just been sitting in the repo and it doesn't sound like there has been any demand for it (that I've heard of, at least).

We can either
1) Remove all the TreeView code
2) Fix/update TreeView code and put it back on display on the site

**Comment by 3xtr4t3rr3str14l** [comment]
I think it could be a "nice to have" to be able to visualize eth2 data structures.  People only expect the site to do serialization/deserialization, so if we add a feature, maybe we'd just need to make an announcement so people would know that they can come to this site for that feature.

**Comment by 3xtr4t3rr3str14l** [comment]
But also, maybe something like this item in this issue (https://github.com/ChainSafe/lodestar/issues/2044) would be better:

> Modified simplified Lodestar nodes that allows users to inspect the beacon state as it is updated. Semi-interactive block explorer based on our node.

**Comment by dapplion** [comment]
Yeah I would focus on Lightclient stuff for now before doing visualization work
